utility/rp_push_client.py

```diff
@@ -0,0 +1,188 @@
+import argparse
+import os
+from concurrent.futures import ThreadPoolExecutor
+from functools import partial
+
+import xmltodict
+from rp_utils.preproc import PreProcClient
+from rp_utils.rp_client import Launch, ReportPortal, RpLog
+from rp_utils.xunit_xml import TestCase, TestSuite, XunitXML
+
+from utility.log import Log
```

Comment:
This is a separate utility and don't see a need to have it integrated with the common log mechanism